src/fun/dice/DiceCommand.ts

```diff
@@ -0,0 +1,54 @@
+import Command from "../../core/Command";
+import { Message, MessageEmbed } from "discord.js";
+import Store from "../../core/Store";
+import Constants from "../../core/Constants";
+
+class DiceCommand extends Command {
+    constructor(store: Store, prefix: string) {
+        super(store, prefix, "dice", 1);
+    }
+
+    checkPermission(msg: Message) {
+        return !msg.author.bot;
+    }
+
+    async execute(msg: Message, args: string[]) {
+        if (args.length === 0) {
+            var rand = Math.floor(Math.random() * 6) + 1;
+            await msg.channel.send("You rolled a " + rand, Constants.black);
+        } else if (args.length === 1 && args[0] === "help"){
+            await msg.channel.send(`Available commands are:
+\`-dice\` - roll a 6-sided dice
+\`-dice <sides>\` - roll a dice with a specific amount of sides
+\`-dice <stuff>\` - choose randomly from a list of comma seperated stuff
+\`-dice <min> <max>\` - choose randomly between numbers in a range
+\`-dice help\` - show this help message`, Constants.black);
```

Comment:
Can you convert this to an embed? (Also this is registering as an error when I compile it in TS, not sure if there's a library version mismatch right now)

Reply:
Okay, done. I don't know much about TypeScript so I may have broken something.

Reply:
I downloaded typescript and ran the build script, and I think I fixed it.